rbkit-lib/subscriber.cpp

```diff
@@ -55,6 +55,12 @@ void Subscriber::triggerGc() {
     commandSocket->sendCommand(triggerGC_Command);
 }
 
+void Subscriber::startCPUProfiling() {
+    RBKit::CmdStartProfile startCPUProfile;
```

Comment:
@emilsoman what is the command which needs to be sent to backend? `start_cpu_profile` ?